devclean.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+  
+# usage instructions
+usage () {
+cat << EOF_USAGE
+Clean the UFS-SRW Application build
+Usage: $0 [OPTIONS] ...
+
+OPTIONS
+  -h, --help
+      show this help guide
+  -a, --all
+      removes bin directory, build directory, and all the submodules
+  --remove
+      removes existing build directories, including those in submodules
+  --clean
+      removes existing build directories and all other build artifacts
+  --build-dir=BUILD_DIR
+      main build directory
+  --bin-dir=BIN_DIR
+      installation binary directory name ("bin" by default; any name is available)
```

Comment:
I think you will need to have `--install-dir` here too, so that you can clean binaries and libraries installed.

Reply:
Thank you for the suggestion @danielabdi-noaa, will add that!

Reply:
@danielabdi-noaa - added the `--install-dir=<INSTALL_DIR>` option, which is current directory (SRW_DIR) by default, and successfully tested it. There is an additional option  to provide a custom name for the binary directory `--bin-dir=<BIN_DIR>`, which is placed to the <INSTALL_DIR>